### PR TITLE
Add Functionality to Log Out

### PR DESCRIPTION
**Pull request by CollinsSpencer**
I added functionality for logging out to the nav bar.

### CODE REVIEW

File: server/logout.php [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?php
+	$root = realpath($_SERVER["DOCUMENT_ROOT"]);
```
Comment (MatthewMeacham) [inline server/logout.php]: Do you think this functionality would be better put in server/callbacks.php? That's where all the login/register/password reset stuff is, so I think it makes sense to keep all this functionality together in that file.

Comment (MatthewMeacham) [reply server/logout.php]: on a sidenote, we should also rename server/callbacks.php I think...

Comment (2mnyzs) [reply server/logout.php]: Agreed on both.

File: components/nav.php [hunk 2]
```diff
@@ -30,7 +30,23 @@
 					<a class="nav-link" href="/queue">Queue</a>
 				</li>
 				<li class="nav-item">
+				<?php if ($USER->isLoggedIn()): ?>
+					<a class="nav-link" onclick="logout()">Log out</a>
+					<script type="text/javascript">
+						function logout() {
+							$.ajax({
+								url : "server/callbacks.php",
```
Comment (MatthewMeacham) [inline components/nav.php]: This actually won't work on any page except the index

File: components/nav.php [hunk 3]
```diff
@@ -30,7 +30,23 @@
 					<a class="nav-link" href="/queue">Queue</a>
 				</li>
 				<li class="nav-item">
+				<?php if ($USER->isLoggedIn()): ?>
```
Comment (MatthewMeacham) [inline components/nav.php]: Also did we figure out why this PHP will fail on the Mac's MAMP?

**Comment by 2mnyzs** [comment]
Works great, LGTM.

File: components/nav.php (hunk 3 above)
Comment (2mnyzs) [reply components/nav.php]: Perhaps it used to be shorthand tags which was an issue for some?

Comment (MatthewMeacham) [reply components/nav.php]: Update, got this fixed by rearranging some of the `require_once` in user.class.php and it ended up working 🤷‍♂️

**Pull request merged**